apps/framework-cli/src/cli.rs

```diff
@@ -250,13 +253,24 @@ async fn top_command_handler(
         } => {
             let run_mode = RunMode::Explicit {};
             info!("Running build command");
-            let project: Project = load_project()?;
-            let project_arc = Arc::new(project);
+            let project_arc = Arc::new(load_project()?);
 
             check_project_name(&project_arc.name())?;
 
             let mut controller = RoutineController::new();
 
+            let route_table = HashMap::<PathBuf, RouteMeta>::new();
+            let route_table: &'static RwLock<HashMap<PathBuf, RouteMeta>> =
+                Box::leak(Box::new(RwLock::new(route_table)));
+            initialize_project_state(project_arc.clone(), &mut *route_table.write().await)
```

Comment:
We are getting rid of `initialize_project_state` with corev2, we probably don't want to add a dependency on there. You could look at the Data models loader to do the same thing.